woocommerce/woocommerce-android
# Shipping labels: Shipping carriers - part 2

This completes the #3525. The unit tests will be added in the next PR.

| ![image](https://user-images.githubusercontent.com/1522856/109868019-7321c200-7c67-11eb-9635-dbc3a8d41679.png) | ![image](https://user-images.githubusercontent.com/1522856/109868050-7b79fd00-7c67-11eb-84c4-d00c18dcd56f.png) |
|-|-|

**To test:**
1. Start the SL creation flow by filling out valid addresses (in the US)
2. Fill out all the steps up to the shipping carrier steps
3. Start the carrier step
4. Notice the rate options are loaded
5. Select a rate
6. Notice the rate layout is expanded with additional options
7. Select the signature option
8. Notice the Update button is show
9. Tap on the Update button
10. Notice the carrier step description is updated with the selected rate
11. Tap on the Edit button
12. Notice the previously selected rate is pre-selected

Update release notes:
